include/sensor_imu_plugin.h

```diff
@@ -74,8 +74,6 @@ class SensorImuPlugin : public SensorPlugin {
   /// Norm of the gravitational acceleration [m/s^2]
   double gravity_magnitude;
 
-  Eigen::Vector3d _gyroscope_bias;
-  Eigen::Vector3d _accelerometer_bias;
```

Comment:
In there a reason that we want to remove this as a member of the imu plugin class? why not initialize it in the constructor?

Reply:
Responded in the wrong location, see comment below:
[https://github.com/PX4/px4-jsbsim-bridge/pull/4#issuecomment-700591201](url)